javascript/imports.md

```diff
@@ -0,0 +1,32 @@
+# Imports
+
+Imports should be sorted alphabetically. That's all.
```

Comment:
Is there a tool for this similar to `goimports` or is this something that developer must manually take care of?

Reply:
For now something the developer must take care of manually. But there are ways to get this into the linter so that an error shows up, and then possibly also into something that does it automatically for us as part of a code style step, that we then add to `make` along with `prettier`.

But for now, keeping it simple.